apps/entity-service/src/entities/processors/project.processor.ts

```diff
@@ -291,9 +289,78 @@ export class ProjectProcessor extends EntityProcessor<
     return pTotal + sTotal + nTotal;
   }
 
-  async loadAssociationData(projectIds: number[]): Promise<number> {
-    const approvedSitesQuery = Site.approvedIdsSubquery(projectIds[0]);
-    const approvedSiteReportsQuery = SiteReport.approvedIdsSubquery(approvedSitesQuery);
-    return await TreeSpecies.visible().collection("tree-planted").siteReports(approvedSiteReportsQuery).sum("amount");
+  async loadAssociationData(projectIds: string[]): Promise<Record<number, ProjectLightDto>> {
+    const associationDtos: Record<number, ProjectLightDto> = {};
+    const numericProjectIds = projectIds.map(id => Number(id)).filter(id => !isNaN(id));
```

Comment:
This method should just accept `number[]` - no reason to strictly cast all the numbers to strings and then parse them back into numbers here.